lib/global_event_technology_api_client/configuration.rb

```diff
@@ -125,7 +125,7 @@ class Configuration
     attr_accessor :force_ending_format
 
     def initialize
-      @scheme = 'https'
+      @scheme = 'http'
```

Comment:
pretty sure we want https

Reply:
Right, good catch! Fixed in [993523c](https://github.com/ramp106/global_event_technology_api_client/pull/3/commits/993523ce71389ca895e42ee19e5d3d0ec7abb462)